regression/contracts/assigns_enforce_literal/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--enforce-contract foo
+^EXIT=1$
+^SIGNAL=0$
+^main.c:\d+:\d+: error: illegal target in assigns clause$
+^CONVERSION ERROR$
+--
+--
+Check that literal constants are rejected from assigns clauses.
```

Comment:
Missing newline.

Check other ".desc" files too.

Reply:
Fixed it.